styles/gemini/catppuccin.user.css

```diff
@@ -0,0 +1,235 @@
+/* ==UserStyle==
+@name Gemini Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/gemini
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/gemini
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/gemini/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Agemini
+@description Soothing pastel theme for Gemini
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('gemini.google.com') {
+  :root body.dark-theme {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  :root body.light-theme {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #rgbify(@color) {
+    @rgb-raw: red(@color), green(@color), blue(@color);
+  }
```

Comment:
Nitpick but can we move this to above the `@catppuccin` palette and right after the final `}` of the main userstyle? Thats how we have it across the rest of the userstyles (mixins at the bottom).

Reply:
Looks like you missed this one @Dandraghas.

Reply:
I didnt miss it, i havent finished it yet, for some reason github automatically requested review